test/assets/podtato-head/initProject.sh

```diff
@@ -10,35 +10,39 @@ case "$1" in
     echo keptn create project "${PROJECT}" --shipyard=./shipyard.yaml   
     keptn create project "${PROJECT}" --shipyard=./shipyard.yaml
     ;;
-  "onboard-service")
-    echo "Onboarding keptn service helloservice in project ${PROJECT}"
-    keptn onboard service helloservice --project="${PROJECT}" --chart=helm-charts/helloserver
+  "create-service")
```

Comment:
do we actually still need this file?

Reply:
good point, I will search if it is used anywhere

Reply:
it's not used, so I removed it. Thanks!